docs/profiles.md

```diff
@@ -177,7 +175,7 @@ For example:
 
     depends:
     - name: gh-profile
-      git: username/project
+      github: username/project
       branch:  desired_branch
```

Comment:
Ok will change this. Looks like this was copied from the `git` section above and I just left it.